src/ui_txn.c

```diff
@@ -451,39 +489,43 @@ struct ux_step {
-  { ALL_TYPES,    "Txn type",       &step_txn_type },
-  { ALL_TYPES,    "Sender",         &step_sender },
-  { ALL_TYPES,    "Fee (uAlg)",     &step_fee },
-  { ALL_TYPES,    "First valid",    &step_firstvalid },
-  { ALL_TYPES,    "Last valid",     &step_lastvalid },
-  { ALL_TYPES,    "Genesis ID",     &step_genesisID },
-  { ALL_TYPES,    "Genesis hash",   &step_genesisHash },
-  { ALL_TYPES,    "Note",           &step_note },
-  { PAYMENT,      "Receiver",       &step_receiver },
-  { PAYMENT,      "Amount (uAlg)",  &step_amount },
-  { PAYMENT,      "Close to",       &step_close },
-  { KEYREG,       "Vote PK",        &step_votepk },
-  { KEYREG,       "VRF PK",         &step_vrfpk },
-  { ASSET_XFER,   "Asset ID",       &step_asset_xfer_id },
-  { ASSET_XFER,   "Asset amt",      &step_asset_xfer_amount },
-  { ASSET_XFER,   "Asset src",      &step_asset_xfer_sender },
-  { ASSET_XFER,   "Asset dst",      &step_asset_xfer_receiver },
-  { ASSET_XFER,   "Asset close",    &step_asset_xfer_close },
-  { ASSET_FREEZE, "Asset ID",       &step_asset_freeze_id },
-  { ASSET_FREEZE, "Asset account",  &step_asset_freeze_account },
-  { ASSET_FREEZE, "Freeze flag",    &step_asset_freeze_flag },
-  { ASSET_CONFIG, "Asset ID",       &step_asset_config_id },
-  { ASSET_CONFIG, "Total units",    &step_asset_config_total },
-  { ASSET_CONFIG, "Default frozen", &step_asset_config_default_frozen },
-  { ASSET_CONFIG, "Unit name",      &step_asset_config_unitname },
-  { ASSET_CONFIG, "Decimals",       &step_asset_config_decimals },
-  { ASSET_CONFIG, "Asset name",     &step_asset_config_assetname },
-  { ASSET_CONFIG, "URL",            &step_asset_config_url },
-  { ASSET_CONFIG, "Metadata hash",  &step_asset_config_metadata_hash },
-  { ASSET_CONFIG, "Manager",        &step_asset_config_manager },
-  { ASSET_CONFIG, "Reserve",        &step_asset_config_reserve },
-  { ASSET_CONFIG, "Freezer",        &step_asset_config_freeze },
-  { ASSET_CONFIG, "Clawback",       &step_asset_config_clawback },
+  { ALL_TYPES,    "Txn type",         &step_txn_type },
+  { ALL_TYPES,    "Sender",           &step_sender },
+  { ALL_TYPES,    "RekeyTo",          &step_rekey },
+  { ALL_TYPES,    "Fee (uAlg)",       &step_fee },
+  { ALL_TYPES,    "First valid",      &step_firstvalid },
+  { ALL_TYPES,    "Last valid",       &step_lastvalid },
+  { ALL_TYPES,    "Genesis ID",       &step_genesisID },
+  { ALL_TYPES,    "Genesis hash",     &step_genesisHash },
+  { ALL_TYPES,    "Note",             &step_note },
+  { PAYMENT,      "Receiver",         &step_receiver },
+  { PAYMENT,      "Amount (uAlg)",    &step_amount },
+  { PAYMENT,      "Close to",         &step_close },
+  { KEYREG,       "Vote PK",          &step_votepk },
+  { KEYREG,       "VRF PK",           &step_vrfpk },
+  { KEYREG,       "Vote first",       &step_votefirst },
+  { KEYREG,       "Vote last",        &step_votelast },
+  { KEYREG,       "Nonparticipating", &step_nonpart },
```

Comment:
I wonder if flipping this to "Participating" (and showing the opposite boolean value) would make this a clearer UI?  "Participating: yes" seems clearer than "Nonparticipating: false".